drivers/sdhc/mcux_sdif.c

```diff
@@ -0,0 +1,463 @@
+/*
+ * Copyright 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_lpc_sdif
+
+#include <zephyr/drivers/sdhc.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/logging/log.h>
+
+
+LOG_MODULE_REGISTER(sdif, CONFIG_SDHC_LOG_LEVEL);
+
+#include <fsl_sdif.h>
```

Comment:
Minor comment, can this be grouped with the rest of the `includes`

Reply:
Fixed, thanks